src/SFA.DAS.EmployerCommitmentsV2.Web/Mappers/Apprentice/ApprenticeshipDetailsRequestToViewModelMapper.cs

```diff
@@ -85,26 +86,35 @@ public async Task<ApprenticeshipDetailsRequestViewModel> Map(ApprenticeshipDetai
                     EmployerReference = apprenticeship.EmployerReference,
                     CohortReference = _encodingService.Encode(apprenticeship.CohortId, EncodingType.CohortReference),
                     EnableEdit = enableEdit,
-                    CanEditStatus = (apprenticeship.Status == ApprenticeshipStatus.Live || 
-                                     apprenticeship.Status == ApprenticeshipStatus.WaitingToStart || 
-                                     apprenticeship.Status == ApprenticeshipStatus.Paused),                    
+                    CanEditStatus = (apprenticeship.Status == ApprenticeshipStatus.Live ||
```

Comment:
ApprenticeshipStatus is set a little further up this mapper, could this logic not just be a property getter? I know this is not something we have changed here. The whitespace changes just highlighted it to me

Reply:
Done